publish_velocity            : true
publish_encoder             : true
publish_current             : false
publish_motor_joint_state   : true
publish_load                : false
update_rate                 : 1000.0
# motor model
motor_nominal_voltage       : 24.0 # Volts
moment_of_inertia           : 0.00001 # kgm^2
armature_damping_ratio      : 0.0001 # Nm/(rad/s)
electromotive_force_constant: 0.03 # Nm/A
electric_resistance         : 13.8 # Ohm
electric_inductance         : 0.001 # Henry
# transmission
gear_ratio                  : 72.0
joint_damping               : 0.0000
joint_friction              : 0.0000
# shaft encoder
encoder_ppr                 : 111
velocity_noise              : 0.0
# PI controller
kp_v                        : 0.02
ki_v                        : 0.06